blackrock/portal/media/js/maps.js

```diff
@@ -152,25 +152,25 @@ if (!Portal.MapMarker) {
                 var infraIcon = infrastructure[0];
                 infraIcon = infraIcon.replace(/ /g, "");
                 infraIcon = infraIcon.replace("-", "");
-                iconUrl = 'http://' + location.hostname + ':' + location.port +
+                iconUrl = 'https://' + location.hostname + ':' + location.port +
```

Comment:
Does this script really need to be constructing full URLs with the protocol, hostname, and port? Wouldn't just using the relative path `/portal/media/...` work just as well and be much simpler?